webtool/test_correct.py

```
@@ -2189,3 +2189,54 @@ def ask_llm_verboten(*a, **kw):
+    Deshalb laeuft hier das ECHTE `_ask_llm`: gefaelscht wird eine Ebene tiefer (`llm.complete`),
+    damit `llm._with_files` die Dateien wirklich oeffnet. Jedes Oeffnen einer `.raw.txt` faellt
+    als Marke ins Protokoll, und `jobs.buche_aktive` sagt, ob die Aufnahme in genau dem Moment
+    gesperrt war.
+    """
+    import builtins
+    from webtool import llm as _llm
+
+    _glossar_projekt(monkeypatch, tmp_path)
+    monkeypatch.setattr(_llm, "use_api", lambda: True)
+    monkeypatch.setattr(_llm, "complete",
+                        lambda *a, **kw: '{"proper_nouns": [], "likely_corrections": []}')
```

Ruff (0.16.2)

[warning] Unused lambda argument: `a`

(ARG005)